### ISSUE TITLE
Cannot set account limits via CLI

### ISSUE THREAD

**Issue by dchristidis** [rucio/rucio#3160]
Motivation
----------

This appears to be related to the introduction of global quota support.

```
> rucio-admin account set-limits dcameron NO-NORGRID-T2_LOCALGROUPDISK 150000000000
'Client' object has no attribute 'set_account_limit'
Rucio exited with an unexpected/unknown error, please provide the traceback below to the developers.
Traceback (most recent call last):
  File "/home/dcameron/dev/ddm/rucio/bin/rucio-admin", line 129, in new_funct
    return function(*args, **kwargs)
  File "/home/dcameron/dev/ddm/rucio/bin/rucio-admin", line 386, in set_limits
    client.set_account_limit(account=args.account, rse=args.rse, bytes=byte_limit)
AttributeError: 'Client' object has no attribute 'set_account_limit'
```

Modification
------------

~~Update `rucio-admin` to use the proper method.~~
Add necessary support to `rucio-admin` but keep `get-limits` and `set-limits` as aliases.

**Comment by davidgcameron** [comment]
This problem also applies to get-limits

```
> rucio-admin account get-limits dcameron CERN-PROD_DATADISK
'Client' object has no attribute 'get_account_limit'
Rucio exited with an unexpected/unknown error, please provide the traceback below to the developers.
Traceback (most recent call last):
  File "/home/dcameron/dev/ddm/rucio/bin/rucio-admin", line 129, in new_funct
    return function(*args, **kwargs)
  File "/home/dcameron/dev/ddm/rucio/bin/rucio-admin", line 400, in get_limits
    limits = client.get_account_limit(account=args.account, rse=args.rse)
AttributeError: 'Client' object has no attribute 'get_account_limit'
```

**Comment by bjwhite-fnal** [comment]
I didn't include a fix for getting the account limits _yet_, as that functionality still seems to be possible with `rucio list-account-limits`.

**Comment by bari12** [comment]
I think David is referencing #3200 
But this can be addressed in a separate PR.

**Comment by dchristidis** [comment]
Thanks for the patch @bjwhite-fnal!

**Comment by ericvaandering** [comment]
Still seeing this in Rucio-admin version 1.21.6. Is there something wrong with our setup?

**Comment by ericvaandering** [comment]
Let me be more clear. There is still no get_account_limit in accountlimitclient

**Comment by bari12** [comment]
Yes, these methods are in the `accountclient`. It is a bit confusing, but I think historically they have always been there.
In `rucio-admin` we are still pointing to the wrong methods though I think. It should query the `get_local_account_limits` and `get_global_account_limits`. @bjwhite-fnal do you perhaps have time to have a look on this?

**Comment by bari12** [comment]
This is partly fixed in @cserf PR #3293 but not fully, since we should still enable the querying of the global limits with `rucio-admin` as well.
